charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.60.0
+version: 0.61.0
```

Comment:
please bump the patch version and not the minor version, the minor version is for when we do an operator release :) thank you!